diff --git a/CMakeLists.txt b/CMakeLists.txt
index c4a7eff..2fb2dfd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,7 +27,6 @@ set(QtWebDAV_SOURCES
 set(QtWebDAV_HEADERS
     qnaturalsort.h
     qwebdav.h
-    qwebdav_global.h
     qwebdavdirparser.h
     qwebdavitem.h
 )
@@ -60,7 +59,6 @@ target_include_directories(QtWebDAV PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
     $<INSTALL_INTERFACE:include>
 )
-target_compile_definitions(QtWebDAV PRIVATE DEBUG_WEBDAV)
 
 # Link Qt libraries
 target_link_libraries(QtWebDAV PUBLIC
